### PR TITLE
WIP: Attestation requests

### PR DESCRIPTION
**Pull request by oed**
This PR adds a simple method for creating attestation requests.

However I think we need to update the [specs for this](https://github.com/uport-project/specs/pull/6) since the request does not specify which identity that should sign the request. So if the user has multiple accounts/identities it's unclear what will happen.

### CODE REVIEW

**Comment by oed** [comment]
Also it might make sense to just put this in the `createRequest method`, but that would enable a dev to make a signing request and a disclosure request at the same time which does not make sense.

**Comment by bdresser** [comment]
@oed looks like the spec has been updated, is this good to merge? (cc @simonas-notcat )

File: src/Credentials.js [hunk 1]
```diff
@@ -102,6 +102,31 @@ class Credentials {
...
+ *
+ *  @example
+ *  const unsignedClaims = [{
+ *    claim: {
+ *      "Citizen of city X": {
+ *        "Allowed to vote": true,
+ *        "Document": "QmZZBBKPS2NWc6PMZbUk9zUHCo1SHKzQPPX4ndfwaYzmPW"
+ *      }
+ *    },
+ *    sub: "2oTvBxSGseWFqhstsEHgmCBi762FbcigK5u"
```
[the remark this answers is not in the document]
Comment (oed) [reply src/Credentials.js]: What is confusing? I think it makes sense.

**Comment by oed** [comment]
@zachferland Updated to match the new spec. I also renamed the method to `createVerificationRequest`.

**Pull request merged**